webapp/components/_new/generic/LabeledButton/LabeledButton.story.js

```diff
@@ -0,0 +1,35 @@
+import { storiesOf } from '@storybook/vue'
+import helpers from '~/storybook/helpers'
+import LabeledButton from './LabeledButton.vue'
+
+storiesOf('Generic/LabeledButton', module)
+  .addDecorator(helpers.layout)
+  .add('default', () => ({
+    components: { LabeledButton },
+    data: () => ({
+      filled: false,
+    }),
+    template: `
+      <labeled-button
+        icon="check"
+        :filled="filled"
+        label="All"
+        @click="filled = !filled"
+      />
+    `,
+  }))
+
+  .add('filled', () => ({
+    components: { LabeledButton },
+    data: () => ({
+      filled: true,
+    }),
+    template: `
+      <labeled-button
+        icon="check"
+        :filled="filled"
+        label="All"
+        @click="filled = !filled"
+      />
+    `,
+  }))
```

Comment:
@mattwr18 cool that you added the story! 💚 

I think, since it's interactive, we don't really need the `filled` story and could instead maybe give the button in the `default` story the label `Toggle me` to make it clear that the button is reacting to input? What do you think?

Reply:
Great idea!